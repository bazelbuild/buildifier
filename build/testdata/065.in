x = 123
y = 0123
z = 0
foo(bar=[0, 01, 02, 3])
