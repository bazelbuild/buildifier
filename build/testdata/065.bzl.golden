x = 123
y = 0o123
z = 0
t = 0x123
foo(bar = [0, 0o0, 0o1, 0o2, 3])
